core/src/main/java/io/questdb/cutlass/text/TextImportExecutionContext.java

```diff
@@ -46,15 +48,19 @@ public AtomicBooleanCircuitBreaker getCircuitBreaker() {
     }
 
     public boolean isActive() {
-        return activeImportId.get() > -1;
+        return activeImportId.get() > NO_IMPORT_ACTIVE;
     }
 
     public boolean sameAsActiveImportId(long importId) {
         return activeImportId.get() == importId;
     }
 
+    public long getCurrentImportId() {
```

Comment:
nit: we should use "active" instead of "current"
```suggestion
    public long getActiveImportId() {
```